gds/src/main/scala/edu/gemini/aspen/gds/configuration/KeywordConfigurationFile.scala

```diff
@@ -0,0 +1,40 @@
+package edu.gemini.aspen.gds.configuration
+
+import edu.gemini.aspen.gds.configuration.KeywordConfigurationItem
+import util.parsing.input.Position
+
+/**
+ * This object provides utility methods to manipulate a configuration file
+ * Adapted from GDSConfigurationFile in previous (actor) version of GDS, with all
+ * file writing/updating removed.
+ */
+object KeywordConfigurationFile {
+  // TODO: This assumes no parsing errors and returns an empty list if there is. There was a
+  // gds-config-validator project that I guess was used for validation. I think we should do
+  // something to force validation, even if it is an exception. Successfully starting with no
+  // warning seems inadvisable, but that may be by design?
+  // We should also, somewhere, make sure that all the types for an epics array are the same.
+  def loadConfiguration(configurationFile: String): KeywordConfiguration = {
+    var results =
+      new KeywordConfigurationParser().parseFileRawResult(configurationFile).getOrElse(Nil)
+
+    //if last line is empty, remove it
+    if (!results.isEmpty && results.last.isEmpty) {
```

Comment:
Should this be handled by the parser?

Reply:
Hmmm. I don't know. This is just a remnant from the original parser. I'll try it and see.

Reply:
Maybe

```scala
val resultsʹ =  if (results.lastOption.exists(_.isEmpty)) results.init else results
```

Reply:
I'll make another pass at the file parsing. I took the 🙈 approach and just pulled the old parser over with as few changes as I could. That probably isn't the best approach.

Reply:
This particular line was only relevant for writing the config out to a file - which we no longer do. So, it will just be removed.